pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -180,6 +180,7 @@ type StandbyDescription struct {
 	GSWalPath   string `json:"gs_wal_path,omitempty"`
 	StandbyHost string `json:"standby_host,omitempty"`
 	StandbyPort string `json:"standby_port,omitempty"`
+	StandbyPrimarySlotName string `json:"standby_primary_slot_name,omitempty"`
```

Comment:
Please use go.fmt on all your files. Alignment is wrong here.

Reply:
Done, it'd be great to have a CI job for this (no one of the current ones was complaining about this)